bigtable/spark/build.sbt

```diff
@@ -18,6 +18,8 @@ name := "bigtable-spark-samples"
 
 version := "0.1"
 
+resolvers += Resolver.mavenLocal
```

Comment:
Why are you asking for Maven Local?  is there another step that loads things into it?

Reply:
Nice catch! It's to speed up dependency resolution when a user uses maven (most likely) or gradle as well as sbt.

The main reason was to let me debug an issue with a dependency that I built locally.

I thought I'd remove it, but decided not to since it may speed things up while resolving dependencies. I'm fine either way.

Reply:
You probably want to remove it, as it will look there anyway.

Reply:
> it will look there anyway.

Nope. It will not (but can) as described in https://www.scala-sbt.org/1.x/docs/Library-Dependencies.html#Resolvers:

> sbt can search your local Maven repository if you add it as a repository